public/less/_dashboard.less

```diff
@@ -26,6 +26,13 @@
                     top: 7px;
                     left: 90%;
                     width: 15%;
+
+					@media only screen and (max-width: 768px) {
```

Comment:
spacing seems to be off

Reply:
Fixed, apparantly my editor was set to use tabs instead of spaces as the rest of the file.